boot/modeenv.go

```diff
@@ -85,10 +92,13 @@ func ReadModeenv(rootdir string) (*Modeenv, error) {
 	if err := cfg.ReadFile(modeenvPath); err != nil {
 		return nil, err
 	}
+
+	fmt.Println(cfg.Options(""))
```

Comment:
Is this println intentional?

Reply:
Dropped

Reply:
no sorry about that, was rushing a bit to get the pr up and missed this